andela/ah-backend-tesseract
# Ch switch to postgesql db 159951958

#### What does this PR do?
Switches the database engine to PostgreSQL.
#### Description of Task to be completed?
- The application should be able to use postgreSQL while in development, testing and production environments
#### How should this be manually tested?
- clone the github repo
- create a databases with names `'ah_database'` and `'ah_test_db'` in postgres.
-  Provide  your database password in `settings.py`
-  cd into the repo and make migrations.
- You should be able to see the tables in your postgres database
#### What are the relevant pivotal tracker stories?
#159951958
